### PR TITLE
replace oraclejdk7 with oraclejdk8

### PR DESCRIPTION
**Pull request by MartinNowak**
- see https://github.com/travis-ci/travis-ci/issues/7964#issuecomment-316769421

Could alternatively use oraclejdk8 IIUC.

### CODE REVIEW

**Comment by akarnokd** [comment]
Oracle/OpenJDK v7 no longer works on Travis. Use `oraclejdk8` instead.

**Comment by MartinNowak** [comment]
> Oracle/OpenJDK v7 no longer works on Travis. Use oraclejdk8 instead.

OpenJDK7 does, only Oracle no longer does, but we can easily update to Oracle JDK8.

**Comment by akarnokd** [comment]
RxJava has been switched to 8 so this project should too. No EC worries after.

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-  - oraclejdk7
+  - oraclejdk8
 
 before_install:
   - chmod +x gradlew
+
+# workaround for missing openjdk7 EC support
+# https://github.com/gradle/gradle/issues/2421#issuecomment-319916874
+install:
+ - JAVA_HOME=$(jdk_switcher home openjdk8) ./gradlew classes testClasses
```
Comment (akarnokd) [inline .travis.yml]: Is this necessary?

Comment (MartinNowak) [reply .travis.yml]: No, this was a leftover from the openjdk7 EC workaround.

**Pull request merged**